stakedex_sdk/tests/test_main.rs

```diff
@@ -71,14 +86,69 @@ fn test_swap_via_stake_unknown_token() {
     assert!(res.is_err());
 }
 
+fn test_swap_via_stake(input_mint: Pubkey, output_mint: Pubkey) {
+    let whale_pk = Pubkey::from_str(WHALE).unwrap();
+    let res = STAKEDEX.quote_swap_via_stake(&QuoteParams {
+        in_amount: 100_000_000_000,
+        input_mint,
+        output_mint,
+    });
+    match res {
+        Err(err) => assert!(err.to_string() == "No route found between pools"),
```

Comment:
```suggestion
        Err(err) => assert!(err == SwapViaStakeQuoteErr::NoRouteFound),
```

`SwapViaStakeQuoteErr` impls `PartialEq` so this should work?

Reply:
errors with `binary operation `==` cannot be applied to type `anyhow::Error``

Reply:
:facepalm: right, I forgot it gets converted to an `anyhow::Error` somewhere in the stack